flink-1.17-connector-bigquery/flink-connector-bigquery/src/main/java/com/google/cloud/flink/bigquery/sink/serializer/AvroSchemaConvertor.java

```diff
@@ -0,0 +1,139 @@
+package com.google.cloud.flink.bigquery.sink.serializer;
```

Comment:
This file is the copy of [AvroSchemaConvertor ](https://github.com/apache/flink/blob/master/flink-formats/flink-avro/src/main/java/org/apache/flink/formats/avro/typeutils/AvroSchemaConverter.java), with slight changes to accomodate `datetime` BigQuery type

Reply:
Note: This class is not simply inherited and methods overwritten as they are private.